docs/releasenotes.md

```diff
@@ -0,0 +1,26 @@
+## HYDROLIB-core 0.5.0 Release Notes
+HYDROLIB-core 0.5.0 mainly adds more support for D-Flow FM 2D3D models in coastal applications.
+
+### Compatibility Notes
+Support for MacOS is temporarily dropped, due to an upgrade to MeshKernel 2.0.2.
```

Comment:
Eventhough it is implied, should we add explicitly that the MeshKernel 2.0.2 does not work with MacOS?

Something like:
`Support for MacOS is temporarily dropped, due to an upgrade to MeshKernel 2.0.2 that currently does not work on MacOS`